packages/react/src/accordion/index.tsx

```diff
@@ -3,12 +3,28 @@ import { CdsAccordionContent as AccordionContent } from '@cds/core/accordion';
 import { CdsAccordionHeader as AccordionHeader } from '@cds/core/accordion';
 import { CdsAccordionPanel as AccordionPanel } from '@cds/core/accordion';
 import '@cds/core/accordion/register';
-import { createComponent } from '../converter/react-wrapper.js';
+import { createComponent } from '@lit-labs/react';
+import * as React from 'react';
 
-export const CdsAccordion = createComponent('cds-accordion', Accordion);
-export const CdsAccordionPanel = createComponent('cds-accordion-panel', AccordionPanel, {
+// add react util here in all components.
+// Document that it needs to be part of each component in React.
+
+export const CdsAccordion = createComponent(React, 'cds-accordion', Accordion);
+export const CdsAccordionPanel = createComponent(React, 'cds-accordion-panel', AccordionPanel, {
   onExpandedChange: 'expandedChange',
   onCdsMotionChange: 'cdsMotionChange',
 });
-export const CdsAccordionHeader = createComponent('cds-accordion-header', AccordionHeader);
-export const CdsAccordionContent = createComponent('cds-accordion-content', AccordionContent);
+export const CdsAccordionHeader = createComponent(React, 'cds-accordion-header', AccordionHeader);
+export const CdsAccordionContent = createComponent(React, 'cds-accordion-content', AccordionContent);
+
+/**
+ * Setting displayName on the prototype is a workaround for now.
+ * See: https://github.com/lit/lit/issues/2154
+ * Follow: https://github.com/lit/lit/pull/2155
+ */
+CdsAccordion.displayName = 'CdsAccordion';
+
+// Setting react version has to be done at component level since we now consume the @lit-labs/react package
+if ((window as any).CDS && !(window as any).CDS._react.version) {
```

Comment:
Checking and adding the react version to the global CDS object for DX.

Reply:
I modified this slightly and removed the (window as any) stuff. Had to make it the way it is for tests to pass.